Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/RoomWrapper.java

```diff
@@ -77,16 +81,23 @@ public void join(PlayerWrapper player, String password) {
     }
 
     public void leave(PlayerWrapper player) {
+        Player unwrapped = player.unwrap();
+
         synchronized (players) {
             PlayerLeftSessionMessage message = new PlayerLeftSessionMessage(player.getPlayerId());
 
             players.remove(player);
             players.stream()
                     .map(PlayerWrapper::getConnection)
                     .forEach(c -> c.send(message));
+
+            if (players.isEmpty()) {
+                room.setHost(null);
+            } else if (room.getHost().equals(unwrapped) && players.size() > 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1155)